ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/GORenderers/Info.plist

```diff
@@ -1,54 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-	<key>CFBundleDisplayName</key>
-        <string>GORenderers</string>
```

Comment:
plist is still needed for iOS deployment. The display name and bundle executable would need to be updated, though